### DISCUSSION TITLE
Ustawienie uBlock

### DISCUSSION THREAD

**Discussion by MESWEB** [MajkiIT/polish-ads-filter#17529] [Q&A] [answered]
![Opera Snapshot_2020-10-31_131818_kccohkcpppjjkkjppopfnflnebibpida](https://user-images.githubusercontent.com/2931024/97780307-caaa0700-1b7b-11eb-9161-74dd3cd16f3d.png)

Czy ktos wie dlaczego moje filtry nie uzywaja wszystkich regul? Widze ze nie wszystkie sa uzyte z widocznych. Mam cos zle ustawione? Pytam bo reklamy z napirpojekt ktore ostatnio zglosilem alej sie pojawiaja.

**Comment by hawkeye116477** [comment, marked as answer]
Po prostu uBO odfiltrowuje duplikaty, czyli już na jakiejś innej liście jest ta sama reguła.

**Comment by xxcriticxx** [comment]
nano defender jest uznawany jako spyware lista do wykasowania

**Reply by hawkeye116477** [reply]
To już zależy czy ma coś z Chromium czy FF.
A sama lista akurat jest bezpieczna, choć bez dodatkowego rozszerzenia raczej nie ma sensu. W przypadku FF można używać obu.
